templates/includes/sidebar.html

```diff
@@ -0,0 +1,9 @@
+<!-- templates/includes/sidebar.html -->
+<div class="sidebar">
+  <ul>
+      <li><a href="#">Home page</a></li>
+      <li><a href="#">Manufacturers</a></li>
+      <li><a href="#">Cars</a></li>
+      <li><a href="#">Drivers</a></li>
+  </ul>
+</div>
```

Comment:
The sidebar is correctly implemented with a list of dummy links as required. This ensures that the sidebar can be included in the base template and provides navigation placeholders for the Home page, Manufacturers, Cars, and Drivers.